Dockerfile

```diff
@@ -11,20 +12,40 @@ RUN apt-get update && \
     libpq-dev \
     curl
 
+# For pyodbc
+RUN curl -s -o microsoft.asc https://packages.microsoft.com/keys/microsoft.asc && \
+    curl -s -o mssql-release.list https://packages.microsoft.com/config/debian/10/prod.list
+RUN apt-get update -y && \
+    apt-get install -y g++ unixodbc-dev
+
 RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
 RUN mv /root/.poetry $POETRY_PATH
 RUN poetry config virtualenvs.create false
 RUN poetry config experimental.new-installer false
 
 COPY poetry.lock pyproject.toml ./
+RUN poetry lock --no-update
 RUN poetry install --no-interaction --no-ansi --no-dev -vvv
 
 
 FROM base as runtime
 
+# For pyodbc
+COPY --from=build microsoft.asc microsoft.asc
+COPY --from=build mssql-release.list mssql-release.list
+
+ENV ACCEPT_EULA=Y
+RUN apt-get update -y && apt-get install -y gnupg2 \
+    && apt-key add microsoft.asc && rm microsoft.asc && \
+    mv mssql-release.list /etc/apt/sources.list.d/mssql-release.list
+RUN apt-get update -y && apt-get install -y unixodbc-bin
```

Comment:
Do we really need to create new layer for each install command and call `apt-get update` every time?